examples/fonty.py

```diff
@@ -74,8 +73,8 @@ def main():
 
     # Get some metrics.
     print("Font metrics for 'Fonty':  %s" % a_sys_font.metrics(text))
-    ch = unicode_("%c") % 0x3060
-    msg = unicode_("Font metrics for '%s':  %s") % (ch, a_sys_font.metrics(ch))
+    ch = str("%c") % 0x3060
+    msg = str("Font metrics for '%s':  %s") % (ch, a_sys_font.metrics(ch))
```

Comment:
this can be further simplified. You can put a character explicitly in the string with \u, and use an f string for the message.